twatbot.py

```diff
@@ -500,11 +500,11 @@ def scrape(user, first_run = False):
 		checkfn = fetch_more_tweets_callback
 		count = -1
 
+	elapsed_time = time.time()
 	insert_pos = 0
-	sys.stdout.write('\rscraping %s... ' % user)
+	sys.stdout.write('\r[%s] scraping %s... ' % (time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(elapsed_time)), user))
```

Comment:
please... a function get_timestamp(date=None) that returns a string
if date is None date = time.time()